applications/crust-explorer.md

```diff
@@ -0,0 +1,127 @@
+# Crust Grant Proposal
+
+- **项目名称: crust-explorer**
+
+- **Team Name:** 草猪科技
+
+- **Payment Address:** crust wallet -
+    5EtKPUiw6BuF8C7EDVVeCY5sAecm3w7vq4vrmgKHFQui7ozB
+
+- **Receivable Token:** 30 CRU
+
+## Overview
```

Comment:
Please use English as applying language

Reply:
> Please use English as applying language

Yep. Since Crust Grants Program has been included in the reference of [Web 3.0 Grants program](https://github.com/w3f/Open-Grants-Program#other-grant-programs) and is now opening to the whole world, application should be submitted in English.

Reply:
ok, we will submit a new English version